defaults/main.yml

```diff
@@ -30,6 +30,7 @@ nextcloud_data_dir: "{{ nextcloud_installation_dir }}/data"
 nextcloud_file_owner: 'www-data'
 # The user that will own nextcloud files.
 
+# IMPORTANT: port and prefix are not supported for nextcloud_version > 19. Use nextcloud_config instead.
```

Comment:
Could we add a check in `tasks/variables.yml` that fails, when the variables `port`and `prefix` are set and the version is >= 20?

Reply:
@docuteam-kun Let me know, if you need help with the check.